openedx/core/djangoapps/credentials/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
Yes it's more like a library. This will be used for validations such as whether user is eligible for credentials, credentials service is enabled or not etc